packages/http-client-java/generator/http-client-generator-test/src/main/java/com/cadl/multipart/models/InheritFileData.java

```diff
@@ -9,9 +9,7 @@
 import com.azure.core.util.BinaryData;
 
 /**
- * Image file of content-type "image/jpeg"
- * 
- * The file details for the "fileData1" field.
+ * Image file of content-type "image/jpeg".
```

Comment:
This diff is also caused by previous `detail` and `description` have difference from current `summary` and `doc`.
previous code-model:
<img width="627" alt="image" src="https://github.com/user-attachments/assets/e401f942-15d5-490f-87d1-b3acd0f52cb7">
current code-model:
<img width="753" alt="image" src="https://github.com/user-attachments/assets/3d23e291-240c-44db-acd2-4b74256510d9">

Reply:
I think we could just accept the change. Would like to hear others' thoughts.